app/code/community/Phoenix/CashOnDelivery/etc/system.xml

```diff
@@ -73,42 +74,47 @@
                             <show_in_store>1</show_in_store>
+                            <depends><enable>1</enable></depends>
                         </title>
                         <order_status translate="label">
-                            <label>New order status</label>
+                            <label>New Order Status</label>
                             <frontend_type>select</frontend_type>
                             <source_model>adminhtml/system_config_source_order_status</source_model>
                             <sort_order>40</sort_order>
                             <show_in_default>1</show_in_default>
                             <show_in_website>1</show_in_website>
                             <show_in_store>1</show_in_store>
+                            <depends><enable>1</enable></depends>
                         </order_status>
                         <sort_order translate="label">
-                            <label>Sort order</label>
+                            <label>Sort Order</label>
                             <frontend_type>text</frontend_type>
                             <sort_order>50</sort_order>
                             <show_in_default>1</show_in_default>
                             <show_in_website>1</show_in_website>
                             <show_in_store>1</show_in_store>
+                            <depends><enable>1</enable></depends>
                         </sort_order>
                         <shippingallowspecific translate="label">
-                            <label>Shipment to applicable countries</label>
+                            <label>Ship to Applicable Countries</label>
                             <frontend_type>allowspecific</frontend_type>
                             <sort_order>60</sort_order>
                             <source_model>adminhtml/system_config_source_payment_allspecificcountries</source_model>
                             <show_in_default>1</show_in_default>
                             <show_in_website>1</show_in_website>
                             <show_in_store>1</show_in_store>
+                            <depends><enable>1</enable></depends>
                         </shippingallowspecific>
                         <shippingspecificcountry translate="label">
-                            <label>Shipment to Specific countries</label>
+                            <label>Ship to Specific Countries</label>
                             <frontend_type>multiselect</frontend_type>
                             <sort_order>70</sort_order>
                             <source_model>adminhtml/system_config_source_country</source_model>
                             <show_in_default>1</show_in_default>
                             <show_in_website>1</show_in_website>
                             <show_in_store>1</show_in_store>
                             <depends>
+                                <enable>1</enable>
```

Comment:
Hiding config data for the payment method when disabled is not something we want to add. Displaying the configuration is often desired even if the method is (temporarily) disabled.